instrumentation/aws-lambda-1.0/library/src/main/java/io/opentelemetry/instrumentation/awslambda/v1_0/ParentContextExtractor.java

```diff
@@ -20,15 +20,15 @@
 
   private static final String AWS_TRACE_HEADER_ENV_KEY = "_X_AMZN_TRACE_ID";
 
-  static Context extract(Map<String, String> headers) {
+  static Context extract(Map<String, String> headers, BaseTracer tracer) {
```

Comment:
This is a bad API, but untangling would add a lot to this PR I think. Would like to address it in the future as tracer APIs are quite unstable anyways.